e2e/npm-ci-registry.ts

```diff
@@ -91,8 +91,11 @@ EOD`;
     fs.removeSync('adduser.sh');
   }
 
-  _registerToCiScope() {
-    this.helper.command.runCmd('npm config set @ci:registry http://localhost:4873');
+  // TODO: improve this to only write it to project level npmrc instead of global one
```

Comment:
should be turned into an harmony issue

Reply:
It's an old issue with our tests. nothing new. it's not related to harmony for my opinion. and I'm not sure it's worth an issue. it will anyway be a low priority and we won't handle it soon.